models/topicsModel.js

```diff
@@ -81,3 +81,24 @@ exports.postCommentToArticle = (newComment, article_id) => {
     return comment
   })
 };
+
+exports.patchArticleById = (article_id, inc_votes) => {
+  if (!article_id || !inc_votes) {
```

Comment:
will there ever not be an article id? Would it manage to even get here if there wasnt an article id, given that its part of the route?